ReactWindows/ReactNative/Views/Text/ReactTextViewManager.cs

```diff
@@ -51,6 +53,76 @@ public void SetSelectable(RichTextBlock view, bool selectable)
         }
 
         /// <summary>
+        /// Sets the TextDecorationLine for the node.
+        /// </summary>
+        /// /// <param name="view">The view.</param>
+        /// <param name="textDecorationLineValue">The TextDecorationLine value.</param>
+        [ReactProp(ViewProps.TextDecorationLine)]
+        public void SetTextDecorationLine(RichTextBlock view, string textDecorationLineValue)
```

Comment:
>SetTextDecorationLine [](start = 20, length = 21)

I think it would be better to ignore this prop for now than to provide an incomplete implementation.  The `textDecorationLine` should be applicable to virtual text nodes (i.e., those that correspond to the inline instances as well).